api.swagger.yaml

```diff
@@ -3753,7 +3757,19 @@ components:
               example: 01234567-89ab-cdef-1234-56789abcdef0
           required:
             - uuid
-    title-uuid-list-common-schema:
+    single-title-uuid-list-common-schema:
```

Comment:
If there's only a single title and uuid in it, in what sense is it a list?

How about something like `title-uuid-schema` and leave the other one with its existing name, but make it `$ref` this one for the contents of the array?

Reply:
Changed it to just a string now. I'm thinking it makes more sense to keep consistent with e.g. manager_uuid as it's a single value reference field.

We don't actually have a person types endpoint yet but will do soon with the taxonomy endpoints work. This should suffice for Sierra club, however.